scripts/prerelease/generate_prerelease_script.py

```diff
@@ -130,10 +130,10 @@ def main(argv=sys.argv[1:]):
         repositories[repo_name] = source_repo
 
     for repo_name, repo_type, repo_url, version in args.custom_repo:
-        if repo_name in repositories:
-            print("The repository '%s' appears multiple times" % repo_name,
+        if repo_name in repositories and repositories[repo_name]:
+            print("custom_repos option overiding '%s' to '%s': '%s' '%s'. " %
+                  (repo_name, repo_type, repo_url, version),
                   file=sys.stderr)
-            return 1
```

Comment:
If you don't break out of the loop with the error, it gets injected into the repositories list in the next 6 lines which is what's used throughout the rest of the script.